packages/ui/src/data/filters/utils.ts

```diff
@@ -0,0 +1,272 @@
+        if (isFilterSelected(selectedFilters, filterGroup, filter.data.key)) {
+            acc.push(filter);
+        }
+        return acc;
+    }, []);
+};
+
+export const getSelectedFilters = (filters: IFilter[], filterGroup: IFilterGroup): IFilter[] => {
+    const selectedFilters = getFiltersQuery();
+    if (isEmpty(selectedFilters)) {
+        return [];
+    }
+
+    switch (filterGroup.type) {
+        case VisualType.Range:
+            return getSelectedFiltersForRange(filters, filterGroup, selectedFilters);
+        default:
+            return getSelectedFiltersOther(filters, filterGroup, selectedFilters);
+    }
+};
+
+export const getRangeSelection = (filters: ISyntheticSqon, filterGroup: IFilterGroup) => {
+    let rangeSelection: IFilterRange = { max: undefined, min: undefined, rangeType: undefined };
+    for (const filter of filters.content) {
+        const filt = filter as IValueFilter;
+        if (filt.content.field === filterGroup.field) {
+            if (filt.op === FieldOperators.between) {
+                rangeSelection = {
+                    ...rangeSelection,
+                    max: filt.content.value[1] as number,
+                    min: filt.content.value[0] as number,
+                };
+            } else {
+                const op = filt.op === FieldOperators['>='] ? 'min' : 'max';
+                rangeSelection = { ...rangeSelection, [op]: filt.content.value[0] };
+            }
+        }
+    }
+
+    return rangeSelection;
+};
+
+export const isFilterSelected = (filters: ISyntheticSqon, filterGroup: IFilterGroup, key: string) => {
+    const isFilterFound = (sqon: ISyntheticSqon, filterGroup: IFilterGroup, key: string): boolean => {
+        if (isReference(sqon)) {
+            return false;
+        } else if (isFieldOperator(sqon)) {
+            const valueContent = (sqon.content as unknown) as IValueContent;
+            return valueContent.value.includes(key) && valueContent.field === filterGroup.field;
+        } else {
+            return sqon.content.reduce(
+                (acc: any, contentSqon: any) => acc || isFilterFound(contentSqon, filterGroup, key),
+                false,
+            );
+        }
+    };
+
+    return isFilterFound(filters, filterGroup, key);
+};
+
+const emptySqon = { content: [], op: BooleanOperators.and };
+
+export const useFilters = () => {
+    let searchParams = new URLSearchParams(window.location.search);
+
+    // @ts-ignore
+    const paramsValues = [...searchParams.values()];
+    // @ts-ignore
+    const filters = paramsValues.length > 0 ? JSON.parse(paramsValues) : emptySqon;
+
+    return { filters };
+};
+
+export const getQueryBuilderCache = (type: string): any => {
+    const items = localStorage.getItem(`query-builder-cache-${type}`);
+
+    if (isEmpty(items)) {
+        return {};
+    }
+
+    try {
+        return JSON.parse(items!);
+    } catch (e) {
+        return {};
```

Comment:
no logging?

Reply:
No need, returning an empty object is what we want